### PR TITLE
chore(letter-size) add letter size option

### PR DESCRIPTION
**Pull request by senechko**
adding Letter Size

### CODE REVIEW

File: src/main/java/com/lob/model/Letter.java [hunk 1]
```diff
@@ -236,6 +239,10 @@ public ZonedDateTime getSendDate() {
         return sendDate;
     }
 
+    public LetterSize getLetterSize() {
+        return letterSize;
+    }
```
Comment (ajpawlicki) [inline src/main/java/com/lob/model/Letter.java:244]: Does this return `null` when the size field doesn't exist?

File: src/test/java/com/lob/model/LetterTest.java [hunk 2]
```diff
@@ -306,6 +307,41 @@ public void testCreateLetterWithMergeVariableObject() throws Exception {
         assertEquals("letter", letter.getObject());
     }
 
+    @Test(expected=InvalidRequestException.class)
```
Comment (ajpawlicki) [inline src/test/java/com/lob/model/LetterTest.java:310]: Should we just add the feature flag to the Java Lib account to test the success case of this?

**Review by ajpawlicki — APPROVED** [review]
LGTM had one suggestion

**Comment by sup** [comment]
Thanks for this!

File: src/main/java/com/lob/model/Letter.java (hunk 1 above)
Comment (senechko) [reply src/main/java/com/lob/model/Letter.java:244]: yes

File: src/test/java/com/lob/model/LetterTest.java (hunk 2 above)
Comment (senechko) [reply src/test/java/com/lob/model/LetterTest.java:310]: do you have an example in another sdk where we do it?

Comment (ajpawlicki) [reply src/test/java/com/lob/model/LetterTest.java:310]: No, looking through our libraries I do not see us do that. I was just wondering if we should test the success case in our ci, but it doesn't seem that common to add flagged features to our libraries in general. This seems fine as is.

**Pull request merged**